src/main/java/org/candlepin/auth/permissions/PermissionFactory.java

```diff
@@ -136,26 +136,59 @@ public PermissionFactory(OwnerCurator ownerCurator) {
     @SuppressWarnings("checkstyle:indentation")
     private Map<String, PermissionBuilder> initBuilders() {
         Map<String, PermissionBuilder> builders = new HashMap<>();
+        builders.put(PermissionType.OWNER.name(), (user, bp, res) -> {
+            Owner owner = res.resolve(bp.getOwner());
+            if (owner == null) {
```

Comment:
nit: For cases like this where we're doing an either/or thing, we can flip the operator and use a ternary op to compress the code down to only a few lines and save a few nanoseconds of execution:

```
return owner != null ?
  List.of(new OwnerPermission(...)) :
  List.of();
```

Reply:
Good call! Updated throughout